CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file
 automatically by Versionist. DO NOT EDIT THIS FILE MANUALLY!
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+# v3.0.0
+## (2022-12-21)
+
+* Bump up base Yocto release layer reference to Langdale [Pablo Saavedra]
```

Comment:
FWIW in Debian the changelog is usually like:
```
  [Pablo Saavedra]
  * change 1
  * change 2
  * change 3

  [Another Contributor]
  * change 1
```

I'm fine with what you have now, just giving an idea in the case that you like it, it allows for example for longer lines.

Reply:
@manuelafm  thanks for the suggestion. In this case, I am using the `balena-versionist` [balena-wpe#semantic-commits](https://github.com/Igalia/balena-wpe#semantic-commits) tool to generate this Changelog. Probably there is way to customize the logs. I chose the default behavior.